### PR TITLE
Updated multi provider docs

### PR DESCRIPTION
**Pull request by DawidSAG**

### CODE REVIEW

File: content/web/libraries-bundle/component-library-ngx.md [hunk 1]
```diff
@@ -65,63 +65,65 @@ A good example to use this concept is the `c8y-action-bar-item` which uses a `ro
...
+* `hookNavigator`: Enables navigator nodes to be shown.
+* `hookAction`: Enables to define the global actions which should be shown or enabled on certain conditions.
+* `hookActionBar`: Enables to define local actions on a page.
+* `hookBreadcrumb`: Can be used to show breadcrumbs in the header bar.
+* `hookSearch`: Allows to define the search to be shown or not.
+* `hookComponent`: A hook to add dynamic components to the UI (e.g. widgets).
+* `hookRoute`: A hook to add a new route.
+* `hookWizard`: A hook to add a entry into a existing wizard.
+* `hookStepper`: A hook to add a step into a existing stepper.
 #### Services {#services}
```
Comment (janhommes) [inline content/web/libraries-bundle/component-library-ngx.md:127]: don't you need to add a break here?

**Review by janhommes — APPROVED** [review]

File: content/web/libraries-bundle/component-library-ngx.md (hunk 1 above)
Comment (DawidSAG) [reply content/web/libraries-bundle/component-library-ngx.md:127]: Current state:
image

Comment (DawidSAG) [reply content/web/libraries-bundle/component-library-ngx.md:127]: It's ok, or should I change it?

**Comment by MWindel** [comment]
@DawidSAG Does this need to go into other release versions? If so, please do the changes in a separate PR.

**Comment by DawidSAG** [comment]
> @DawidSAG Does this need to go into other release versions? If so, please do the changes in a separate PR.

@MWindel After discussing with Jan, this PR should be sufficient.

**Pull request merged**